### DISCUSSION TITLE
Table lines text size

### DISCUSSION THREAD

**Discussion by manny4566** [grafana/grafana#68433] [Dashboards, Panels & Visualizations]
Hi,

do somebody know how i can change the size of the text in a table?

The problem has occurred since the last update. with the change to 9.5.2, the text no longer fits. Before the text was displayed correctly and there was no line break.
In addition, I can not understand the formatting. 
Because in other columns with the same character length this does not happen.
for examle see image:
Thanks

![grafik](https://github.com/grafana/grafana/assets/23473485/8a3fb582-d4a0-43a2-bfba-b59fdb161598)

![grafik](https://github.com/grafana/grafana/assets/23473485/0683c59a-4f91-4d71-ab7d-b26b3c0e79f6)

**Comment by gustl-kiwi** [comment]
I would also appreciate a font size setting for the table. Sometime the text is just to small

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). As such, I will be closing this discussion.

In later versions of Grafana, there is a "Text Size" setting that will let you change the size of the title and value. You can also manipulate the bar size. If those options are not sufficient, feel free to make an [issue](https://github.com/grafana/grafana/issues) if  you would like to request the feature.
